community/xkcdpass/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
+# Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
+pkgname=xkcdpass
+pkgver=1.9.0
+pkgrel=0
+pkgdesc="Generate secure multiword passwords/passphrases"
+url="https://github.com/redacted/XKCD-password-generator"
+arch="noarch"
+license="BSD"
+depends="py3-xkcdpass"
```

Comment:
Pardon, this comment should be for `subpackages="py3-$pkgname:py3"`, not this line.  And also I was too hurry, this actually may not be entirely bad solution. I must think about it for a while.